statemachine.py

```diff
@@ -41,19 +46,29 @@ def fsm():
     asr.subscribe("nao")  # pepper start to listens, eyes turns blue
 
     while True:
+        if state == '6':            # start gif at start of breathing exercise
+            tablet.showImage(url)
+        elif state == '8':          # stop gif at end of breathing
+            tablet.hideImage(url)
+
         tts.say(script[state]['content'].encode('ascii', 'ignore'))  # convert from unicode to ascii for compatibility
         time.sleep(5)           # delay to allow user time to reply
+        
         response = memory.getData("WordRecognized")  # retrieve response
         print('response: ' + str(response) + '\n')
         print('state: ' + state)
+        
         feedback = is_positive(response[-2], positive, negative)    # sort response
+        
         if feedback == -1 or state == '-1':  # Detect if the user says 'quit'
             break  # Break from loop
         try:
             state = script[state]['next'][feedback]  # switch state of conversation
         except IndexError:
             state = script[state]['next'][0]    # for if only one state listed in "next"
 
+        previousResponse = response[:]         # save the previous entry, could be useful for detecting if user has not responded
```

Comment:
It  seems this line is useless for now?

Reply:
Yeah it does nothing, that was from when I thought response would be a list of all responses in the conversation not overwritten.